modules/account-lib/test/unit/coin/avaxc/transactionBuilder/transfer.ts

```diff
@@ -153,4 +153,25 @@ describe('Avax C-Chain Transfer Transaction', function () {
       (e) => e.message === 'Transfers can only be set for send transactions',
     );
   });
+
+  it('Should build non signed transfer with final v ', async function () {
+    initTxBuilder();
+    txBuilder
+      .transfer()
+      .amount('100000000000000000') // This represents 0.1 Avax = 0.1 Ether
+      .contractSequenceId(1)
+      .expirationTime(50000)
+      .to(testData.TEST_ACCOUNT_2.ethAddress)
+      .key(testData.OWNER_2.ethKey);
+    const tx = await txBuilder.build();
```

Comment:
[np] do this in one go

```
    const tx = await txBuilder
      .transfer()
      .amount('100000000000000000') // This represents 0.1 Avax = 0.1 Ether
      .contractSequenceId(1)
      .expirationTime(50000)
      .to(testData.TEST_ACCOUNT_2.ethAddress)
      .key(testData.OWNER_2.ethKey)
     .build();
```

Reply:
It can't be together. TransferBuilder doesn't has build. 
https://github.com/BitGo/BitGoJS/blob/e7f5982373ab52d69c67cc15abfe303c52860247/src/coin/eth/transactionBuilder.ts#L371

Reply:
weird ok